.github/workflows/markdown-links.yml

```
@@ -0,0 +1,19 @@
+name: markdown-links
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
```

copied from [deeprank2](https://github.com/DeepRank/deeprank2/blob/main/.github/workflows/markdown-link-check.yml):

```suggestion
on:
  push:
    branches:
      - main
    paths:
      # filetypes
      - "**.md"
      - "**.rst"
      # self
      - ".github/workflows/markdown-link-check.yml"
  pull_request:
    types: [opened, synchronize, reopened, ready_for_review]
    paths:
      # filetypes
      - "**.md"
      - "**.rst"
      # self
      - ".github/workflows/markdown-link-check.yml"```